tests/phpunit/elementor/includes/widgets/test-tabs.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Includes\Widgets;
+
+use Elementor\Plugin;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Tabs extends Elementor_Test_Base
+{
+	public function test_widget_display_in_panel__depends_on_nested_elements_experiment() {
+		$tabs= Plugin::instance()->widgets_manager->get_widget_types( 'tabs' );
+		$this->assertFalse( $tabs->show_in_panel() );
```

Comment:
I think the way to test this properly isn't by making a function to test the experiment... I think the more accurate way of testing it is by creating an instance of the element, getting the element config (`Element::get_config()`) and checking the `'show_in_panel'` property value.
This should be done before and after de-activating the experiment.

Reply:
this is exactly what get_widget_types does 
it will init the widgets ( if not performed already ) and then assign an instance o the array, the get_widget_types (Name) returns an instance if it exists